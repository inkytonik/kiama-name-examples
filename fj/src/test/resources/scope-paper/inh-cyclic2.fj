// inheritance/cyclic.spt:one-step inheritance cycle
class A extends B {
  A() { super(); }
}
class B extends A {
  B() { super(); }
}
