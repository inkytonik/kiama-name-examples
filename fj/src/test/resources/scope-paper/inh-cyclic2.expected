src/test/resources/scope-paper/inh-cyclic2.fj:2:1:error: superclass cycle from A
class A extends B {
^
src/test/resources/scope-paper/inh-cyclic2.fj:5:1:error: superclass cycle from B
class B extends A {
^
